lib/bundle/checker.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+module Bundle
+  module Checker
+    class Base
+      PACKAGE_TYPE = :pkg
+      PACKAGE_TYPE_NAME = "Package"
```

Comment:
Are these variables ever used? If not they could be removed or made a comment.

Reply:
They are not used, mostly there for catching poor inheritance. I guess I'll make them a comment.